docs/connection-and-channel-management.md

```diff
@@ -0,0 +1,53 @@
+# Connection and channel management
+
+Spaces SDK uses the Ably Core SDK client [connections](https://ably.com/docs/connect?lang=javascript) and [channels](https://ably.com/docs/channels?lang=javascript) to provide higher level features like cursors or locations. Both connections and channels will transition through multiple states throughout their lifecycle. Most state transitions (like a short loss in connectivity) will be handled by the Ably SDK, but there will be use cases where developers will need to observe these states and handle them accordingly.
+
+This document describes how to access a connection and channels on Spaces, and where to find information about how to handle their state changes.
+
+## Connection
+
+When initializing the Spaces SDK, an Ably client is passed as a required argument:
+
+```ts
+const client = new Realtime.Promise({ key: "<API-key>", clientId: "<client-ID>" });
+const spaces = new Spaces(client);
+```
+
+The Spaces instance exposes the underlying connection which is an event emitter. It can be used to listen for changes in connection state. The client and connection are both available on the Spaces instance:
+
+```ts
+spaces.client.connection.on('disconnected', () => {})
```

Comment:
Out of curiosity, what is the reason to expose the same thing via two different objects?

Reply:
One is client and other the connection - the connection, as per @mattheworiordan comment is for convenience https://github.com/ably-labs/spaces/pull/110#discussion_r1297078614, as that's part your most likely to interact with

Reply:
I saw that @dpiatek but he suggested exposing `spaces.connection.on()` and `spaces.client`. My question is why are we also exposing `spaces.client.connection.on()`

Reply:
in my understanding, that was the intent - exposing the whole interface, not particular methods